server/useragent/useragent.go

```diff
@@ -0,0 +1,19 @@
+package useragent
+
+import (
+	"context"
+	"runtime"
+
+	"github.com/kubernetes-incubator/cri-o/version"
+)
+
+// Get is the User-Agent the CRI-O daemon uses to identify itself.
+func Get(ctx context.Context) string {
+	httpVersion := make([]VersionInfo, 0, 6)
```

Comment:
Should this be a constant?

Reply:
Mmm what do you mean? This is just initializing the array (and I need to fix it also)

Reply:
Ok  I did not know I saw the 6 and thought that was related to version.